cmd/sonobuoy/app/utils/mode/mode.go

```diff
@@ -0,0 +1,99 @@
+package mode
```

Comment:
duplicate package 📦 📦

Reply:
I'd also be ok dropping utils and adding mode/ as a directory under app/, but up to you